src/webgpu/web_platform/reftests/README.txt

```diff
@@ -8,7 +8,6 @@ This tests things like:
 - The canvas renders with the correct transfer function.
 - The canvas blends and interpolates in the correct color encoding.
 
-TODO(#915): Test all possible output texture formats (currently only needs rgba16float).
 TODO(#916): Test all possible ways to write into those formats (currently only testing B2T copy).
```

Comment:
unrelated but I just noticed this is outdated
```suggestion
TODO(#916): canvas_complex: Test all ways to write into textures (currently only testing copy methods).
```

Reply:
Done!